hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/OzoneDelegationTokenSecretManager.java

```diff
@@ -441,10 +441,12 @@ public boolean verifySignature(OzoneTokenIdentifier identifier,
       signerCert = getCertClient().getCertificate(
           identifier.getOmCertSerialId());
     } catch (CertificateException e) {
+      LOG.error("getCertificate with identifier {} failed", identifier, e);
       return false;
     }
 
     if (signerCert == null) {
+      LOG.error("signerCert is null");
```

Comment:
Should this message also indicate which certificate was not found?

```suggestion
      LOG.error("signerCert is null for serialId {}", identifier.getOmCertSerialId());
```

Reply:
Thank you @adoroszlai. Yes, I believe it should be helpful to log this information as well.